crypto/stark/src/rational_expression.rs

```diff
@@ -133,44 +153,56 @@ impl RationalExpression {
     /// Calculates an upper bound. Cancelations may occur.
     // Note: We can have trace polynomials of different degree here if we want.
     pub fn degree(&self, trace_degree: usize) -> (usize, usize) {
-        self.degree_impl(1, trace_degree)
+        self.degree_impl(1, trace_degree, &[]) // TODO for autogen
     }
 
     pub fn trace_degree(&self) -> (usize, usize) {
-        self.degree_impl(0, 1)
+        self.degree_impl(0, 1, &[]) // TODO for autogen
     }
 
     // TODO: do this with a generic function.
-    fn degree_impl(&self, x_degree: usize, trace_degree: usize) -> (usize, usize) {
+    fn degree_impl(
```

Comment:
Do we use this before the claim is substituted?

Reply:
On the rust side, we do not. I think we will need to call this function for the autogen to work though.